contracts/rbn-staking/IncentivizedVotingLockup.sol

```diff
@@ -60,12 +71,13 @@ contract IncentivisedVotingLockup is
 
   /** Lockup */
   uint256 public globalEpoch;
-  uint256 public totalLocked;
```

Comment:
remove `totalLocked` bc its only utility is caching for gas savings in redeemer (since we only have to pull totalLocked() not get the actual balance)